tests/casp/oci_install.pm

```diff
@@ -16,21 +16,56 @@ use base "y2logsstep";
 use testapi;
 use caasp;
 
+sub settings_not_lost {
+    send_key 'alt-p';    # partitioning
+    assert_screen 'prepare-hard-disk';
+    send_key 'alt-b';    # back
+    assert_screen 'oci-overview-filled';
+    send_key 'alt-b';    # booting
+    assert_screen 'inst-bootloader-settings';
+    send_key 'alt-c';    # cancel
+    assert_screen 'oci-overview-filled';
+    send_key $cmd{next};    # network configuration
+    assert_screen 'inst-networksettings';
+    send_key $cmd{next};    # next
+    assert_screen 'oci-overview-filled';
+    send_key 'alt-k';       # kdump
+    assert_screen 'inst-kdump';
+    send_key 'alt-o';       # OK
+    assert_screen 'oci-overview-filled';
+}
+
 sub run() {
-    send_key $cmd{install};
+    my $abort = 2;          # repeat once
+    while ($abort--) {
+        # poo#16408 part 1
+        settings_not_lost if $abort;
 
-    # Accept simple password
-    handle_simple_pw;
+        send_key $cmd{install};
 
-    # Accept update repositories during installation
-    if (check_var('REGISTER', 'installation')) {
-        assert_screen 'registration-online-repos';
-        send_key "alt-y";
-    }
+        # Accept simple password
+        handle_simple_pw;
+
+        # Accept update repositories during installation
+        if (check_var('REGISTER', 'installation')) {
+            assert_screen 'registration-online-repos';
+            send_key "alt-y";
+        }
 
-    # Confirm installation start
-    assert_screen "startinstall";
-    send_key $cmd{install};
+
+        # Confirm installation start
+        assert_screen "startinstall";
+
+        # poo#16408 part 2
+        if ($abort) {
+            send_key 'alt-b';    # abort
+            assert_screen 'oci-overview-filled';
+            set_var 'SIMPLE_PW_CONFIRMED', 0;    # to handle_simple_pw again
```

Comment:
Removing set_var 'SIMPLE_PW_CONFIRMED' should work now as SR is merged in new ISO:
https://build.suse.de/request/show/128853